README.md

```diff
@@ -1,2 +1,33 @@
-# hw01
-Repository for first CFSS homework assignment
+# Homework 01: Edit README.md
+## HyungJin Cho
+  [//]: # (Comment: #~###### is a header tag <h1>~<h6>)
+
+Brief Biography | GitHub Workflow
+--- | ---
+In this first section, I'll introduce myself using Markdown | In this second section, I'll report my process on GitHub
+  [//]: # (Comment: --- | --- creates a table)
+
+### Brief Biography
+
+Hello, I'm HyungJin.  
+  [//]: # (Comment: Newline by two spaces at the end of the line)
```

Comment:
Heh, I never figured out how to add comments in markdown documents. Looks like the solution presented [here](http://stackoverflow.com/questions/4823468/comments-in-markdown). Just be sure not to **over-document** your code. That is once you start writing programs in R, you don't need to have a comment for every line of code. You want to find the right balance between documenting your work and running code in discrete chunks. Too many comments might make it difficult for outsiders to review your code.